<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDocFamilyForm</class>
 <widget class="QWidget" name="CreateDocFamilyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
               color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
              }
QTabWidget {
               color: blue;
               padding: 4px;
               border-radius: 5px;
               border: 1px solid grey; 
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QWidget {
               color: rgb(32, 74, 135);
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QScrollBar {
			   background-color: rgb(240, 240, 240);
              }
QLabel {
               color: rgb(32, 74, 135);
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 11pt &quot;MS Shell Dlg 2&quot;;
              }
QLineEdit {
               color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QComboBox {
			   color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
               font: 9pt &quot;MS Shell Dlg 2&quot;;
			   background-color: rgb(255, 255, 255);
              }
QTextEdit {
               color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QSpinBox {
               color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }
QListView {
               color: black;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
	           font: 10pt &quot;MS Shell Dlg 2&quot;;
              }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="3" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="addCategoryButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
          </property>
          <property name="text">
           <string>Add category</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="backButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
                color: darkblue;
               padding: 4px;
               border: 1px solid rgb(0, 85, 255); 
               border-radius: 5px;
               border-style: outset;
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: white;
               border: 1px solid grey;
               background-color: rgb(0, 78, 235);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
               }</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="submitButton">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
               color: white;
               padding: 4px;
               border: 1px solid grey; 
               border-radius: 5px;
               border-style: outset;
               max-width: 120px;
               background-color: rgb(0, 85, 255);
               font: 10pt &quot;MS Shell Dlg 2&quot;;
              }

QToolButton:hover {
               color: darkblue;
               border: 1px solid grey;
               background-color: rgb(255, 255, 255);
               }

QToolButton:pressed {
               color: rgb(0, 0, 197)
               background: shadow;
               border-style: inset;
			    background-color: rgb(0, 78, 235);
               }</string>
          </property>
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>300</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>798</width>
           <height>298</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="5" column="3">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="10">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Search text </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="11">
           <widget class="QLineEdit" name="familyNameLine"/>
          </item>
          <item row="4" column="8">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>K.P. 5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Description </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="11">
           <widget class="QTextEdit" name="familyDescrText"/>
          </item>
          <item row="4" column="7">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>K.P. 4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>K.P. 1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Family Description</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>K.P. 2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Family Name</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Category </string>
            </property>
           </widget>
          </item>
          <item row="4" column="6">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>K.P. 3</string>
            </property>
           </widget>
          </item>
          <item row="5" column="10">
           <widget class="QLineEdit" name="searchTextLine"/>
          </item>
          <item row="5" column="4">
           <widget class="QLineEdit" name="kp_1_Line">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QLineEdit" name="kp_2_Line">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="7">
           <widget class="QLineEdit" name="kp_4_Line">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="descriptionline"/>
          </item>
          <item row="5" column="0">
           <widget class="QLineEdit" name="categoryLine"/>
          </item>
          <item row="5" column="6">
           <widget class="QLineEdit" name="kp_3_Line">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="8">
           <widget class="QLineEdit" name="kp_5_Line">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="5" column="9">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel {
               color: darkblue;
               padding: 4px;
               border-radius: 5px;
               border-style: outset;
	           font: 12pt &quot;MS Shell Dlg 2&quot;;
              }</string>
     </property>
     <property name="text">
      <string>CREATE DOCUMENT FAMILY</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
